### PR TITLE
use new osg-release with non-mirrored goc repos

### PR DESCRIPTION
**Pull request by DrDaveD**
For now I am using it from koji-hub because it isn't released yet.

### CODE REVIEW

**Comment by DrDaveD** [comment]
@bbockelm Please review.  I cannot merge until you approve.

**Comment by DrDaveD** [comment]
@brianhlin: @bbockelm has not been able to get around to reviewing this.  This was put into production yesterday and I want to merge it and get it tagged.  Please review.

**Review by brianhlin — APPROVED** [review]
LGTM, merge as you see fit

**Pull request merged**
